### ISSUE TITLE
Useless use of cat award :star:

### ISSUE THREAD

**Issue by evandrix** [BBVA/kapow#118]
README.md: `cat /var/log/apache2/access.log | grep 'File does not exist' | kapow set /response/body`

should be: `grep 'File does not exist' /var/log/apache2/access.log | ...`

**Comment by nilp0inter** [comment]
You are right, the `cat` that we have in the README.md is useless, but we put it there on purpose :) .

The README's goal is to show, in the minimal amount of space to not very savvy Unix users, that this line is, in fact, a script; and you can pipe things together.  We selected `grep` and `cat` because are very basic Linux commands.
